core/base/document.php

```diff
@@ -1172,6 +1174,33 @@ public static function on_import_replace_dynamic_content( $config, $map_old_new_
 		return $config;
 	}
 
+	/**
+	 * On import replace dynamic content.
+	 *
+	 * @since 3.8.0
+	 * @access public
+	 *
+	 * @param array $config
+	 * @param array $new_ids_map
+	 * @param array|null $controls
+	 *
+	 * @return array Element data.
+	 */
+	public static function updated_on_import_replace_dynamic_content( array $config, array $new_ids_map, $controls = null ) : array {
```

Comment:
Maybe use $data or any other general naming?
So in the future, we can use this function for more diverse uses.

```suggestion
	public static function updated_on_import_replace_dynamic_content( array $config, array $data, $controls = null ) : array {
```

Reply:
Changed